scripts/build_deb.sh

```diff
@@ -103,21 +105,24 @@ done
 unattended_upgrades_files=("51algorand-upgrades")
 mkdir -p "${PKG_ROOT}/etc/apt/apt.conf.d"
 for f in "${unattended_upgrades_files[@]}"; do
-    cp "installer/${f}" "${PKG_ROOT}/etc/apt/apt.conf.d"
+    < "installer/${f}" \
+      sed -e "s,@CHANNEL@,${CHANNEL}," \
```

Comment:
This is a way I've enjoyed expanding bash variables in templates in the past that you may like better. With this approach you can use bash style syntax to include values.

```
eval "cat <<EOF
$(<file.tmpl.txt)
EOF
" > file.txt
```

Reply:
I'm not sure how that would help here.

For context, this block was flagged for an unnecessary use of `cat`, which has since been removed, replaced by the statement that you see here.  I personally like that rule (UUOC) and  would also like to avoid `eval` unless necessary.

Reply:
I should add that the intention of this PR was just to implement the changes needed to support the repos.  A larger change such as the one you're suggesting could be its own PR (in other words, I'm not against that, just don't want to do it here).

Reply:
Hmm there may be a way to do this without cat. The reason this can be more powerful is that you don't have to do a sed individually for env vars you want to evaluate. A case could be made that in some cases defining fields you want to evaluate can be better. I'm usually more inclined to go with generalized templating strategies, but it's more of a preference thing.

Reply:
Yes, it's a valid point, but I still think that that would be a larger change not really related to the intent of this PR and should be its own PR.